EpikCord/ext/tasks.py

```diff
@@ -0,0 +1,41 @@
+import asyncio
+import typing
+
+# time constants in seconds
+MINUTE = 60
+HOUR = MINUTE * 60
+DAY = HOUR * 24
+WEEK = DAY * 7
+
+
+class Task:
```

Comment:
This is a weird to for a first feature. I would have wait for a more solid base before adding back something like this.

Reply:
This is just a rewrite, the tasks ext is good. It isn't reliant on a "solid base", it's an individual thing,